### PR TITLE
Add microk8s takeover

### PR DESCRIPTION
**Pull request by shivjoshi1996**
## Done

- Added "A" variant of  microk8s takeover
- Added takeover to the cycle

## TODO

- Add "B" variant takeover
- Add UTM links & CTA links

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check it looks like the [design](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/168#issuecomment-443182549)

## Issue / Card

Fixes: #4383

### CODE REVIEW

**Comment by anasereijo** [comment]
- the image and text are not vertically aligned. please add `u-vertically-center` to the image
- should we have a full stop after the h4 copy?
- the suru background should be always fixed to the right side.  `background-size: cover;`
- left align image on iPad Pro, see below:

screenshot 2018-12-05 at 10 14 20

**Comment by anasereijo** [comment]
@anthonydillon nice!
When resizing the browser I get a white line on the left side. Not sure if it's just on my screen or something you can fix.

screenshot 2018-12-05 at 12 35 51

File: templates/takeovers/_microk8s-takeover.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<section class="p-strip--image is-dark is-deep js-takeover p-takeover--microk8s">
+  <div class="row u-equal-heights">
```
Comment (shivjoshi1996) [inline templates/takeovers/_microk8s-takeover.html]: Shouldn't this be "u-equal-height"? (No s)

Comment (anthonydillon) [reply templates/takeovers/_microk8s-takeover.html]: Good spot :+1:

**Pull request merged**